proto/panacea/datapool/v2/pool.proto

```diff
@@ -9,22 +9,14 @@ import "google/protobuf/duration.proto";
 
 // Pool defines a data pool
 message Pool {
-  uint64                    pool_id = 1;
-  uint64                    round = 2;
-  repeated string           data_schema = 3;
-  uint64                    target_num_data = 4; // target number of data
-  uint64                    cur_num_data = 5;    // current number of data
-  uint64                    max_nft_supply = 6;  // max supply of data access NFT of the pool
-  uint64                    num_issued_nft = 7;  // current issued NFT
-  cosmos.base.v1beta1.Coin  nft_price = 8;
-  repeated string           trusted_data_validators = 9;
-  repeated string           trusted_data_issuers = 10;
-  cosmos.base.v1beta1.Coin  deposit = 11;
-  google.protobuf.Duration  download_period = 12 [
-    (gogoproto.stdduration) = true
-  ];
-  string                    curator = 13;
-  string                    status = 14;
+  uint64     pool_id = 1;
+  string     pool_address = 2;
+  uint64     round = 3;
+  PoolParams pool_params = 4;
```

Comment:
I wonder what the criteria for separating Pool and PoolParams are.
As expected, it seems that the input value and the state are divided.

Reply:
Exactly. I divided the input values as a separate struct. 
Just pass inputs to `keeper.CreatePool` and make new pool in `keeper.CreatePool`.

Reply:
Yeah, It seems to good.